src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -339,11 +339,13 @@ private Settings createJwtAuthenticatorSettings(Settings settings) {
         settingsBuilder.put(jwtSettings);
 
         if (jwtSettings.get("roles_key") == null && settings.get("roles_key") != null) {
```

Comment:
What happen when roles_key is null? I know its not a part of this CR, so just wanted to check?

Reply:
if roles_key is null we default to using "roles"

Reply:
so basically roles_key is clear, if nothing is specified, we don't use roles from the token/SAMLResponse.